README.md

```diff
@@ -22,25 +22,22 @@ Cylc is used to encode the entire pipeline from start to finish and relies on th
     * this will start a compute session for 1 day with 32 GB memory and 20 cores
     * see [here](https://docs.ccv.brown.edu/oscar/submitting-jobs/interact) for more options
 
-3. Load the Julia module
-   - [ ] `module load julia/1.9.0`
-
-4. Build a virtual environment and install Cylc
+3. Build a virtual environment and install Cylc
    - [ ] `cd <your-project-path>/ice-floe-tracker-pipeline`
    - [ ] `conda env create -f ./config/ift-env.yaml`
    - [ ] `conda activate ift-env`
 
-5. Register an account with [space-track.org](https://www.space-track.org/) for SOIT
+4. Register an account with [space-track.org](https://www.space-track.org/) for SOIT
 
-6. Export SOIT username/password to environment variable
+5. Export SOIT username/password to environment variable
    - [ ] From your home directory `nano .bash_profile`
    - [ ] add `export HISTCONTROL=ignoreboth` to the bottom of your .bash_profile
         * this will ensure that your username/password are not stored in history
         * when exporting the following environment variables, there __must__ be a space in front of each command
    - [ ] ` export SPACEUSER=<firstname>_<lastname>@brown.edu`
    - [ ] ` export SPACEPSWD=<password>`
```

Comment:
From Josh's talk yesterday, I was thinking maybe secrets could be used for this. Not sure if it's possible.